mobile/src/board/BoardPages.tsx

```diff
@@ -0,0 +1,29 @@
+import { useParams } from "react-router-dom";
+
+import BoardView from "./components/BoardView";
+import { useBoard } from "./BoardHooks";
+import { boardRepository } from "./BoardRepository";
+
+export function GetById() {
+  const { id } = useParams();
+  const [board, error] = useBoard(id!);
+
+  const rotateRemainingTile = async () => {
+    boardRepository.rotateRemainingTile(id!);
+  };
+
```

Comment:
Why is this here instead of in `BoardView` directly ? Besides, FYI, the convention for callbacks is to name the prop `onEvent` and the handler `handleEvent`. Here that would be `onRotateRemainingTile` and `handleRemainingTile`

Reply:
If you wanted to keep the repository logic outside `BoardView`, please just rename the prop and handler

Reply:
I wanted to keep the repository logic outside the `BoardView` , I will rename the props and handlers